xchange-blockchain/src/main/java/org/knowm/xchange/blockchain/BlockchainAuthenticated.java

```diff
@@ -0,0 +1,88 @@
+package org.knowm.xchange.blockchain;
+
+import org.knowm.xchange.blockchain.dto.BlockchainException;
+import org.knowm.xchange.blockchain.dto.account.*;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+
+/**
+ * @see <a href="https://api.blockchain.info/v3/#/">Swagger</a>
+ */
+@Path("v3/exchange")
+@Produces(MediaType.APPLICATION_JSON)
+public interface BlockchainAuthenticated extends Blockchain {
+
+    /**
+     * Receive current account balances
+     *
+     * @return
```

Comment:
Here you have two options: 
1. Put what you are returning.
2. Remove return at all.
(Same comment for all methods)